Dockerfile.production

```diff
@@ -1,7 +1,11 @@
-FROM python:3.8-slim
-
+FROM python:3
 ENV PYTHONUNBUFFERED 1
-RUN apt-get update && apt install -y libgdal32 wait-for-it
+
+RUN echo "Installing GDAL dependencies" && \
+    apt-get update && apt install -y libgdal20 wait-for-it && \
+    echo "Install C library for geoip2" && \
+    apt-get install -y libmaxminddb0 libmaxminddb-dev mmdb-bin
+
```

Comment:
@Xpirix need to also add this

https://github.com/qgis/qgis-feed/blob/master/Dockerfile#L27C1-L29C62

Reply:
@dimasciput , Okay, added it